src/BigFish/PaymentGateway/Request/Init.php

```diff
@@ -282,6 +282,14 @@ class Init extends RequestAbstract
 	 */
 	public $moduleVersion;
 
+	/**
+	 * Proxy address
+	 *
+	 * @var string
+	 * @access public
+	 */
+	public $gatewayProxy;
```

Comment:
I think, you don't need to create this class variable, because the IP address will be checked for every API request, not just for Init.